opentracing/ext/tags.go

```diff
@@ -0,0 +1,49 @@
+package ext
+
+import (
+	"github.com/opentracing/api-golang/opentracing"
+)
+
+var (
+	// RPCTargetService records target service name for an RPC call
+	RPCTargetService = &stringTag{"rpc.target"}
+
+	// RPCHostname records the host name of the target service
+	RPCHostname = &stringTag{"rpc.hostname"}
+
+	// RPCHostIPv4 records IP v4 host address of RPC target service
+	RPCHostIPv4 = &uint32Tag{"rpc.ipv4"}
+
+	// RPCHostIPv6 records IP v6 host address of RPC target service
+	RPCHostIPv6 = &stringTag{"rpc.ipv6"}
+
+	// RPCPort records port number of RPC target service
+	RPCPort = &uint16Tag{"rpc.port"}
+)
+
+type stringTag struct {
+	Key string
+}
+
+// Add adds a string tag to the `span`
+func (tag *stringTag) Add(span opentracing.Span, value string) {
```

Comment:
per the above, we could have something like `RPCSourceAnnotations` with an `Add(span, peerHostname, peerService, peerIPv4, peerIPv6, peerPort)` sort of signature. no strong feelings, just an idea.